### ISSUE TITLE
open source? 

### ISSUE THREAD

**Issue by alexlib** [jonnyhigham/PTVResearch#2]
Hi, 

is the code open sourced? I'm not an expert of Matlab apps - don't know if the code is open and readable. 

Would you consider joining the @openpiv group? see please www.openpiv.net

**Comment by jonnyhigham** [comment]
Hi Alex, 
I'd be happy to join the OpenPIV group, I've previously followed your work with the 3D PTV, something I was aiming on extending this work to. 

So, the toolbox is built in Matlab's new AppDesigner. I have included the source code which means you can then open the code and play with it. 

Thanks, 
Jonny

**Comment by alexlib** [comment]
great. you're welcome to join us on the Google forum were we can discuss how to merge/combine/use in synergy your code with the existing openpiv code base. 

I don't know how to see the code of your App on Github - apparently this new Mathworks format is not recognized as XML, but it basically it just that. The best part of Git is the use of source code :) How do you edit the code in Matlab? 

https://groups.google.com/forum/#!forum/openpiv-users

**Comment by alexlib** [comment]
We actually have two groups, OpenPIV is for everything like PIV, including 2D PTV. and there is another group that does 3D-PTV (Lagrangian 3D turbulence is the main interest), called OpenPTV Of course you are welcome to join both. 

www.openpiv.net
www.openptv.net 
(not very original :)

**Comment by jonnyhigham** [comment]
I don't think it will open. As XML, this is the next generation of GUIDE Matlab used to use. To look at the code, in Matlab type AppDesigner and it will then launch, and you should be able to open the source code.
